src/transport/tests/LoopbackTransportManager.h

```diff
@@ -27,26 +27,26 @@ class LoopbackTransportManager
 {
 public:
     /// Initialize the underlying layers.
-    CHIP_ERROR Init()
+    static CHIP_ERROR Init()
```

Comment:
Could you change the name of this function, if it needs to be static. By convention ([almost all](https://github.com/search?q=repo%3Aproject-chip%2Fconnectedhomeip+%22static+CHIP_ERROR+Init%28%29%22&type=code)) `Init()` functions are not static in whole project. Making this `Init()` as static might confuse someone...

Reply:
Fixed; LoopbackTransportManager is no longer a singleton, instead it's a static member of LoopbackMessagingContext